spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController do
+
+  let(:video) { create(:video) }
+  let(:user) { create(:user) }
+
+  describe 'GET #new' do
+    it "renders the :new view"do
+    get :new, :video_id => video.id
+    expect(response).to render_template("new")
+    end
+  end
+
+  describe 'GET index' do
+    it "assigns @comments" do
+      comment = video.comments.create(content: "yolo")
+      get :index, :video_id => video.id
+      expect(assigns(:comments)).to eq([comment])
+  end
+
+    it "renders the index template" do
+      get :index, :video_id => video.id
+      expect(response).to render_template("index")
+    end
+  end
+
+  describe 'POST create' do
+    it "creates a new comment" do
+      session[:user_id] = user.id
+      post :create, {:video_id => video.id, comment: {content: "yolo4lyfe"}, user_id: user.id}
+      assert_response :redirect
```

Comment:
You are asserting that the redirect happened here, but not that the comment got created. You might want to split into two tests, one for each, or test both in one, which would make some folks frown. In either case, the description should match what you test.